src/Compilers/CSharp/Portable/Compilation/BuiltInOperators.cs

```diff
@@ -204,6 +205,11 @@ internal void GetSimpleBuiltInOperators(UnaryOperatorKind kind, ArrayBuilder<Una
                     // No built-in operator true or operator false
                     ImmutableArray<UnaryOperatorSignature>.Empty,
                     ImmutableArray<UnaryOperatorSignature>.Empty,
+                    GetSignaturesFromUnaryOperatorKinds(new []
```

Comment:
>GetSignaturesFromUnaryOperatorKinds(new [] [](start = 20, length = 42)

I think we should revert all changes to this file, it feels like an overkill and too much churn for a simple case that can be handled specially, similar to the ```..```.